src/BreakfastSerializer/Plugin/PluginNotFoundException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace BDBStudios\BreakfastSerializer\Plugin;
+
+
+class PluginNotFoundException extends \Exception
```

Comment:
Also would it be better to extend `InvalidArgumentException`?

http://php.net/manual/en/class.invalidargumentexception.php